src/components/ItemRecordDetailPane/ItemRecordDetailPane.tsx

```diff
@@ -22,98 +22,57 @@ export const ItemRecordDetailPane: React.FC<IItemRecordDetailPane> = ({ detailPa
-                    <>
-                      {selectedItem?.id}
-                    </>
-                    )
-                  }
+                <KeyValue label={<FormattedMessage id='ui-workflow.workflows.detail.item.id' />}>
+                  { Boolean(selectedItem?.id) && (<>{ selectedItem?.id }</>) }
                 </KeyValue>
               </Col>
               <Col xs={6}>
-                <KeyValue label={<FormattedMessage id="ui-workflow.workflows.detail.item.active" />}>
+                <KeyValue label={<FormattedMessage id='ui-workflow.workflows.detail.item.active' />}>
                   {
-                    Boolean(selectedItem?.active) && (
-                    <>
+                    Boolean(selectedItem?.active) && (<>
                       { selectedItem?.active ?
-                        <FormattedMessage id="ui-workflow.workflows.detail.item.active.true" /> :
-                        <FormattedMessage id="ui-workflow.workflows.detail.item.active.false" />
+                        <FormattedMessage id='ui-workflow.workflows.detail.item.active.true' /> :
+                        <FormattedMessage id='ui-workflow.workflows.detail.item.active.false' />
                       }
-                    </>
-                    )
+                    </>)
                   }
                 </KeyValue>
               </Col>
             </Row>
             <Row>
               <Col xs={6}>
-                <KeyValue label={<FormattedMessage id="ui-workflow.workflows.detail.item.deployment-id" />}>
-                  {
-                    Boolean(selectedItem?.deploymentId) && (
-                    <>
-                      { selectedItem?.deploymentId }
-                    </>
-                    )
-                  }
+                <KeyValue label={<FormattedMessage id='ui-workflow.workflows.detail.item.deployment-id' />}>
+                  { Boolean(selectedItem?.deploymentId) && (<>{ selectedItem?.deploymentId }</>) }
                 </KeyValue>
               </Col>
               <Col xs={6}>
-                <KeyValue label={<FormattedMessage id="ui-workflow.workflows.detail.item.history-time-to-live" />}>
-                  {
-                    Boolean(selectedItem?.historyTimeToLive) && (
-                    <>
-                      { selectedItem?.historyTimeToLive }
-                    </>
-                    )
-                  }
+                <KeyValue label={<FormattedMessage id='ui-workflow.workflows.detail.item.history-time-to-live' />}>
+                  { Boolean(selectedItem?.historyTimeToLive) && (<>{ selectedItem?.historyTimeToLive }</>) }
                 </KeyValue>
               </Col>
             </Row>
             <Row>
               <Col xs={6}>
-                <KeyValue label={<FormattedMessage id="ui-workflow.workflows.detail.item.set-up" />}>
+                <KeyValue label={ <FormattedMessage id='ui-workflow.workflows.detail.item.set-up' /> }>
```

Comment:
Do we want consistent formatting within the curly brackets? Some have spaces and some do not.

Reply:
I don't think we have an established practice.
I've had to guess at the practice.
I can fix this one.